### ISSUE TITLE
"History" list item context menu is not accessible using keyboard

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#311]
**Repro Steps:**
* Launch Calculator app.
* Navigate to History list item and try to open the context menu
* Observe the issue.

**Actual Result**
History list item context menu is not accessible using keyboard. It is accessible using mouse.

**Expected Result**
History list item context menu should be accessible using keyboard.

App Version: 10.1902.16.0

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by sonali9696** [comment]
Could you please elaborate on what is "history context menu"? I know of only one history symbol which is at the top right of the calculator which is accessible from the keyboard by pressing ctrl+H

![history](https://user-images.githubusercontent.com/12599774/54862160-82e6bb00-4d5c-11e9-8669-8b517ce862e2.png)

**Comment by grochocki** [comment]
Right-clicking on a history item will bring up a context menu. This should be accessible with keyboard.

![image](https://user-images.githubusercontent.com/1414423/54890787-2244a480-4e68-11e9-9e7b-b49f92f233d9.png)

**Comment by rudyhuyn** [comment]
It usually happens when the context menu is associated to the `ItemTemplate` instead of the 
`ItemContainer` (not really intuitive), I will fix that.
